owncloudApp/src/main/java/com/owncloud/android/ui/activity/PrivacyPolicyActivity.java

```diff
@@ -100,12 +101,10 @@ public void onReceivedError(WebView view, int errorCode, String description, Str
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
         boolean retval = true;
-        switch (item.getItemId()) {
-            case android.R.id.home:
-                finish();
-                break;
-            default:
-                retval = super.onOptionsItemSelected(item);
+        if (item.getItemId() == android.R.id.home) {
+            finish();
+        } else {
+            retval = super.onOptionsItemSelected(item);
```

Comment:
I know that this name comes from time ago but what does `retval` mean?  Could we use a more descriptive name?

Reply:
TBH, I don't know what it means...